src/Services/Cda/CdaTemplateImportDispose.php

```diff
@@ -1228,21 +1228,21 @@ public function insertImportedUser($value, $create_user_name = false)
         $userName = "";
 
         if (!empty($value['provider_fname'])) {
-            $value['provider_name'] = $value['provider_fname'] ?: 'External';
-            $value['provider_family'] = $value['provider_lname'] ?: 'Provider';
+            $value['provider_name'] = $value['provider_fname'] ?? 'External';
+            $value['provider_family'] = $value['provider_lname'] ?? 'Provider';
```

Comment:
This changes the logic. There's a difference between a null and empty string when null coalescing. $value mostly won't ever be undefined to coalesce to null so if $value is empty 'External' string will not populate. The change will break logic.

So this was not a mistake. It shouldn't throw any warnings either. To reiterate: if $value = "" your changes will return "" and not 'External'

Reply:
Or if warnings it could be written
`$value['provider_name'] = ($value['provider_fname'] ?? '') ?: 'External';`

Reply:
Warnings happen with elvis if variable is not set. Yep, figured these changes may do something and got a headache trying to work out difference between elvis and null coelescing https://www.fparedes.com/blog/null-coalescing-vs-elvis-operator-php/ :)

Completely agree with your suggestion and will make the changes :)